docs/modules/ROOT/pages/cops_rails.adoc

```diff
@@ -3181,6 +3181,42 @@ scope :something, -> { where(something: true) }
 | Array
 |===
 
+== Rails/ShortI18n
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Pending
+| No
+| Yes (Unsafe)
+| 2.7
+| -
+|===
+
+This cop enforces that short forms of `I18n` methods are used:
+`I18n.t` instead of `I18n.translate` and `I18n.l` instead of `I18n.localize`.
+
+=== Examples
+
+[source,ruby]
+----
+# bad
+I18n.translate :key
+translate :key
+I18n.localize Time.now
+localize Time.noew
```

Comment:
I'm worried about false positives for `translate` without a receiver.
I think this cop can provide two options, `conservative` and `aggressive`.

- `EnforcedStyle: conservative (default)` targets `I18n.translate` and `I18n.localize`. 
- `EnforcedStyle: aggressive` targets `I18n.translate`, `translate`, `I18n.localize`, and `localize`.

I think it will make the cop safe by default.